### ISSUE TITLE
TV4play: option -A  does no longer work

### ISSUE THREAD

**Issue by bartdedecker** [spaam/svtplay-dl#1442]
### svtplay-dl versions:

% svtplay-dl --version
svtplay-dl 4.9-5-g77b43ea

### Operating system and Python version:

Windows 10
Python 3.8.0

### What is the issue:

KeyError in tv4play.py, line 107
      107    program = janson2[f"Program:{show}"]

% svtplay-dl --verbose -u LOGIN -p PASSWORD -A --get-only-episode-url https://www.tv4play.se/program/morden-i-sandhamn
DEBUG [1637145962.640686] D:\a\svtplay-dl\svtplay-dl\lib\svtplay_dl\utils\getmedia.py/get_media: version: 4.9-5-g77b43ea
DEBUG [1637145962.6456845] D:\a\svtplay-dl\svtplay-dl\lib\svtplay_dl\service\__init__.py/__init__: service: tv4play
DEBUG [1637145962.6456845] D:\a\svtplay-dl\svtplay-dl\lib\svtplay_dl\utils\http.py/request: HTTP getting 'https://www.tv4play.se/program/morden-i-sandhamn'
DEBUG [1637145962.6486816] C:\hostedtoolcache\windows\Python\3.9.8\x64\lib\site-packages\urllib3\connectionpool.py/_new_conn: Starting new HTTPS connection (1): www.tv4play.se:443
DEBUG [1637145963.1078277] C:\hostedtoolcache\windows\Python\3.9.8\x64\lib\site-packages\urllib3\connectionpool.py/_make_request: https://www.tv4play.se:443 "GET /program/morden-i-sandhamn HTTP/1.1" 200 None
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.9.8\x64\Lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 113, in run
  File "C:\hostedtoolcache\windows\Python\3.9.8\x64\Lib\site-packages\cx_Freeze\initscripts\Console.py", line 15, in run
  File "bin\svtplay-dl", line 7, in <module>
  File "D:\a\svtplay-dl\svtplay-dl\lib\svtplay_dl\__init__.py", line 69, in main
  File "D:\a\svtplay-dl\svtplay-dl\lib\svtplay_dl\utils\getmedia.py", line 61, in get_media
  File "D:\a\svtplay-dl\svtplay-dl\lib\svtplay_dl\utils\getmedia.py", line 79, in get_all_episodes
  File "D:\a\svtplay-dl\svtplay-dl\lib\svtplay_dl\service\tv4play.py", line 107, in find_all_episodes
KeyError: 'Program:morden-i-sandhamn'

**Comment by bartdedecker** [comment]
If you change line 107 in tvplay.py into:

program = janson2["Program:{\"nid\":\"" + show + "\"}"]

it works again.

**Comment by spaam** [comment]
thanks for looking into how to fix it ,  i will commit a fix tomorrow and release a new version with it :)

**Comment by suncore** [comment]
> If you change line 107 in tvplay.py into:
> 
> program = janson2["Program:{"nid":"" + show + ""}"]
> 
> it works again.

That fix gives me:
```
    program =  janson2["Program:{"nid":"" + show + ""}"] # janson2[f"Program:{show}"]
                                  ^
SyntaxError: invalid syntax

```

**Comment by suncore** [comment]
I figured it out eventually :-)
This works:
`program = janson2['Program:{"nid":"' + show + '"}']`

**Comment by anisqetch** [comment]
It does not work for me.
But this works:
program = janson2['Program:{"nid":' + '"' + f"{show}" + '\"}']

**Comment by bartdedecker** [comment]
Yes, I know. The webform ate the backslashes.
Bart

**Comment by hereinmyhead** [comment]
@spaam When do you think you will release this fix? I'm kind of miss the -A option for TV4 Play. :) Thank you!

**Comment by JohnnyJorgensen** [comment]
Hej @spaam - is there anything new about this problem? Like hereinmyhead, I miss the -A option... :-)
Merry Christmas and thanks a lot for all your work with SVTPlay by the way! (y)

**Comment by spaam** [comment]
this is fixed in 4.10 and its out now. bit late but yeah :)
